### ISSUE TITLE
setting the http2.maxHeaderListPairs

### ISSUE THREAD

**Issue by hugebdu** [grpc/grpc-node#2431]
Is there any way to set the [`maxHeaderListPairs`](https://nodejs.org/api/http2.html#http2createserveroptions-onrequesthandler) for the gRPC server [here](https://github.com/grpc/grpc-node/blob/master/packages/grpc-js/src/server.ts#L401)?

we have a use case related to gradual rollout, when the number of headers gets much higher than the default 128.

Thanks!

**Comment by murgatroid99** [comment]
No, there is currently no way to set that option in the gRPC API.

**Comment by hugebdu** [comment]
I can try submitting a PR.
Checked java and go grpc impls, both do support this option.

**Comment by galElmalah** [comment]
@murgatroid99 Can we advance with a PR?
